src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,95 @@
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const personSlug = useParams();
+  const isSelected = (slug: string) => {
+    return personSlug.slug === slug;
+  };
+
+  const parents = (parentName: string | null, parentRole: string) => {
+    const parent = people.find((person) => person.name === parentName);
+    let className = '';
+
+    if (parent) {
+      if (parentRole === 'mother') {
+        className = 'has-text-danger';
+      }
```

Comment:
Why mothers? It should be red for all the women.
Also, you can move this logic into PersonLink component
![image](https://user-images.githubusercontent.com/96409563/213274351-2f5448eb-823c-403c-b08e-55998dddc419.png)